tests/Integration/Framework/BindingInterfacesInGacelaConfigFile/IntegrationTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GacelaTest\Integration\Framework\BindingInterfacesInGacelaConfigFile;
+
+use Gacela\Framework\Gacela;
+use GacelaTest\Integration\Framework\BindingInterfacesInGacelaConfigFile\LocalConfig\Domain\AbstractFromAnonymousClass;
+use GacelaTest\Integration\Framework\BindingInterfacesInGacelaConfigFile\LocalConfig\Domain\AbstractFromCallable;
+use GacelaTest\Integration\Framework\BindingInterfacesInGacelaConfigFile\LocalConfig\Domain\InterfaceFromAnonymousClass;
+use GacelaTest\Integration\Framework\BindingInterfacesInGacelaConfigFile\LocalConfig\Domain\InterfaceFromCallable;
```

Comment:
Could we come up with a better naming for this module name instead of `LocalConfig`?

Reply:
As we agreed, we'll create a follow-up PR renaming this directory, because it's not the only one but some other integration tests have the same "copy/paste" problem, so we'll refactor them all at once because they are all related due to the same reason.